test/user/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '2.4'
+
+services:
+  alpine:
+    image: alpine:3.17.3
+    tty: true 
+    volumes:
+      - app_data:/tmp
+    labels:
+      - docker-volume-backup.archive-pre.user=testuser
+      - docker-volume-backup.archive-pre=/bin/sh -c 'whoami'
+
+
+  backup:
+    image: offen/docker-volume-backup:${TEST_VERSION:-canary}
+    deploy:
+      restart_policy:
+        condition: on-failure
+    environment:
+      BACKUP_FILENAME: test.tar.gz
+      BACKUP_CRON_EXPRESSION: 0 0 5 31 2 ?
+      EXEC_FORWARD_OUTPUT: "true"
+    volumes:
+      - archive:/archive
```

Comment:
I know this is ported from the other test case, but in theory, this could also just be `./local:/archive` which would allow you to just look up the file in `./local` in the test case and you don't need to do Docker voodoo to find out about the volume location.

You can check this test case for an example:https://github.com/offen/docker-volume-backup/blob/main/test/ownership/docker-compose.yml

Reply:
Done